Library/Homebrew/brew.rb

```diff
@@ -88,7 +88,7 @@ def require?(path)
   end
 
   if internal_cmd
-    Homebrew.send cmd.to_s.tr("-", "_").downcase
+    Homebrew::InstallCommand.send cmd.to_s.tr("-", "_").downcase
```

Comment:
I'd suggest declaring a method in the Homebrew module called `commands` that calls `Homebrew::InstallCommand.commands` for now.